script.module.neverwise/addon.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.neverwise" name="NeverWise Kodi toolkit" version="1.0.9" provider-name="NeverWise">
+<addon id="script.module.neverwise" name="NeverWise Kodi toolkit" version="1.1.1" provider-name="NeverWise">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
-    <import addon="script.module.beautifulsoup" version="3.2.1"/>
+    <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+    <import addon="script.module.dateutil" version="2.4.2"/>
   </requires>
   <extension library="lib" point="xbmc.python.module"/>
   <extension point="xbmc.addon.metadata">
     <platform>all</platform>
-    <summary lang="it">Strumenti per gli addons audio/video di Kodi.</summary>
-    <summary lang="en">Toolkit for Kodi audio/video addons.</summary>
-    <description lang="it">Questi strumenti sono utilizzati negli addons per Kodi sviluppati da NeverWise.</description>
-    <description lang="en">These toolkit are used in the addons for Kodi developed by NeverWise.</description>
+    <summary lang="it_IT">Strumenti per gli addons audio/video di Kodi.</summary>
```

Comment:
I don't think this works for gotham (only helix and above). Do not forget you're submitting to the gotham branch so that is the lowest version you intend to support. However, script modules do not show anywhere in the kodi interface so this is not really a problem. Can you do a quick test in gotham to confirm there are no problems?